<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>file_entry_widget</class>
 <widget class="QWidget" name="file_entry_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1634</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Entry</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="filename_check">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="legend_check">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Legend</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Colormap:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="colormap_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>jet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>viridis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>winter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>autumn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spring</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>summer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blues</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reds</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greys</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greens</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Oranges</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Purples</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Vg Format:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="vg_format_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Voltage sweep</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arbitrary</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Sweep:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="sweep_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>hysteresis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>forward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>backward</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Current:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="current_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>ids</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>igs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>both</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Vd region:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="vd_region_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>both</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>neg</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X Scale:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="x_scale_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>log</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y Scale:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="y_scale_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>log</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="absolute_value_check">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Absolute Value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="position_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="duplicate_button">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>➕​</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="remove_button">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>❌</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
